proto/cluster.proto

```diff
@@ -155,6 +155,12 @@ message Cluster {
 
   // Output. The status to show the cluster status.state
   string cluster_state = 9;
+
+  // Output. The service endpoint of the cluster
+  map<string, string> service_endpoint = 10;
```

Comment:
em.. Can we split PR into two? One for service_endpoint and one for event support